### ISSUE TITLE
Pingback notification bugs

### ISSUE THREAD

**Issue by elibud** [wordpress-mobile/WordPress-iOS#9480]
### Expected behavior
1. I should be able to see the correct avatar in a pingback notification detail screen.
2. I should be able to easily tap on the URL where the pingback is coming from to visit the post.

### Actual behavior
1. The avatar and its name are lost between the notification list and the notification detail screen.
![image uploaded from ios 6](https://user-images.githubusercontent.com/5558824/40621716-4f87c75a-6274-11e8-946f-5017f12498a1.png)|![image uploaded from ios 5](https://user-images.githubusercontent.com/5558824/40621717-4fb52d94-6274-11e8-9b78-05e457331940.png)
2. The area where I can tap on the post title to visit the site is too small. 

### Steps to reproduce the behavior

Get a pingback notification (compose a post sharing a link to one of your existing posts) and once you get the notification check the avatar on the details and try to tap on the right hand side of the link.
 
Related to: https://github.com/wordpress-mobile/WordPress-iOS/issues/9479

##### Tested on iPhone X, iOS 11.3, WPiOS 10.0

**Comment by elibud** [comment]
@jleandroperez this type of notification seems to be using `NoteBlockCommentTableViewCell`, but it probably should have its own `TableViewCell` type, what do you think?

**Comment by jleandroperez** [comment]
@elibud It *really* depends on how it should look (or... say, how it looks currently on the web). Do you happen to have access to that sample notifcation? (a screenshot would do!).

**Comment by elibud** [comment]
None of the bugs is present on the web version.

**Comment by jleandroperez** [comment]
@elibud no, but i mean: can you please please paste a screenshot of the web version of the notification? (just wanna figure out if we need to fix the AttributedString generator, or we need a whole different UI)

**Comment by elibud** [comment]
Sure, here it is:

![j3](https://user-images.githubusercontent.com/5558824/40630700-3b95e060-62ab-11e8-9967-7abf1d48607c.png)

**Comment by jleandroperez** [comment]
@elibud i *think* the link might be a `Range` not being processed (and thus, the blog name does not get the NSLink attribute). This _should_ be a quick fix.

The UI looks pretty much like a Comment's Body (all of the Comment Actions + Gravatar + Blog URL). Unless we  need to tweak / tailor down the Pingback UI, we probably don't need another cell type.

I'll take a peek during the Groundskeeping hackweek, thanks for the screenshots Eli!!

**Comment by elibud** [comment]
That's good news, I'll leave this one with you then :) 
I have another ticket to improve the overall design of this https://github.com/wordpress-mobile/WordPress-iOS/issues/9479 so it's OK to just fix the bugs on this one.

**Comment by mattmiklic** [comment]
I got slightly different results from what Eli got above when I tried testing this. The article name, site name, and excerpt all came through correctly, and they match what I see in Calypso (down to the lack of an avatar).

<img width="805" alt="screen shot 2018-05-29 at 4 07 28 pm" src="https://user-images.githubusercontent.com/1396552/40683200-cc3f376e-635b-11e8-8d00-4ffedc2b1d5f.png">

![img_1752](https://user-images.githubusercontent.com/1396552/40683206-d41fa7ac-635b-11e8-8af8-28d7d99de1c3.PNG)

If all those are consistent, I think the main design issue to be solved in #8479 is to make it more obvious that there's a link that will open the full post.
